ibl_to_nwb/metadata/ecephys.yml

```
@@ -82,3 +82,7 @@ Ecephys:
       description: Proportion of quality metrics passed.
     - name: allen_location
       description: Brain region reference in the Allen Mouse Brain Atlas.
+    - name: uuid
```

```suggestion
    - name: local_id
```

@grg2rsr How about this then?

The thing is, 'uuid' is not at all descriptive since there are - within these files anyway - 3 or more fields that are in actuality, a UUID of some kind

So using a more readable name is in everyones best interest

---

agreed and done